### ISSUE TITLE
make use of incremental VCS build

### ISSUE THREAD

**Issue by rmarquis** [rmarquis/pacaur#291]
From [Allan McRae's blog](http://allanmcrae.com/2014/12/pacman-4-2-released/):

```
Incremental VCS Builds: Previously makepkg would remove its working copy of the VCS source directory before starting a new build. Now makepkg will just update the source copy (or attempt to in the case of SVN…) and build the package. This brings VCS builds in line with those using non-VCS sources. A new option -C/--clean was added to makepkg to remove the old $srcdir before building for cases where incremental builds fail.
```

Check if the makepkg -o/-e trick to rebuild the VCS packages only if necessary still makes sense with this change. Also, check if we can make use of that new makepkg -C flag.

**Comment by rmarquis** [comment]
With the last changes in pacaur 4.2.16 and 4.2.17, integrity check has been moved up and the makepkg -o/-e trick has been removed, as it broke `--devel`.

Currently, `--devel --needed` doesn't work correctly anymore. All packages are built before being skipped. Investigate and fix.

**Comment by rmarquis** [comment]
The `--devel --needed` command has been fixed in c352e4164802638a992fd2d92fc8abd0e4b1ac36 and 9bcf7ce9bf31287b55b97bb5732bccd1c71bc4d0.
